cpp/src/strings/regex/regexec.cpp

```diff
@@ -144,7 +130,7 @@ void reprog_device::destroy() { delete this; }
 
 std::size_t reprog_device::working_memory_size(int32_t num_threads) const
 {
-  return relist::alloc_size(_insts_count, num_threads) * 2;
+  return compute_working_memory_size(insts_counts(), num_threads);
```

Comment:
Are these parameters backwards?
```suggestion
  return compute_working_memory_size(num_threads, insts_counts());
```

Reply:
Good catch!